src/Microsoft.Azure.WebJobs.Host/Extensions/IJobHostMetadataProvider.cs

```diff
@@ -60,5 +60,12 @@ public interface IJobHostMetadataProvider
         /// </summary>
         /// <param name="output">The <see cref="TextWriter"/> to write the graph to.</param>
         void DebugDumpGraph(TextWriter output);
+
+        /// <summary>
+        /// Gets function metadata.
+        /// </summary>
+        /// <param name="methodName">Method name.</param>
+        /// <returns></returns>
+        MethodMetadata GetMethodMetadata(string methodName);
```

Comment:
>methodName [](start = 48, length = 10)

We should comment in the interface whether this methodName match is case sensitive or not.

Reply:
Also, we should comment what we do if the method name doesn't exist. (throw? return a null MethodMetadata)  . The current impl looks like it always create an object. 

---
In reply to: [148619644](https://github.com/Azure/azure-webjobs-sdk/pull/1405#discussion_r148619644) [](ancestors = 148619644)